kartothek/io/dask/bag.py

```diff
@@ -288,3 +292,25 @@ def build_dataset_indices__bag(
             update_indices_from_partitions, dataset_metadata_factory=ds_factory
         )
     )
+
+
+def collect_dataset_statistics(store, dataset_uuid, table_name, predicates=None, sample_size=0.3):
+    """
+    WIP: Retrieve some statistics, maybe even plot them, generate a report...?
+    """
+    # TODO not sure if this is the right place for this function
```

Comment:
It returns a dataframe, so technically speaking this should live in the `io.dask.dataframe` module by current convention